components/Layout.js

```diff
@@ -14,7 +14,7 @@ export default function Layout({ children, theme, toggleTheme }) {
         <>
           <StyledHeader>
             <span>PawConnect</span>
-            <ThemeToggler onClick={toggleTheme}>
+            <ThemeToggler onClick={toggleTheme} aria-label="Toggle theme">
```

Comment:
I am not the aria Expert, but maybe you will be a little more specific about what is happening here.